database/sql/V13__ALTER_STAZIONI.sql

```diff
@@ -1 +1,8 @@
-alter table NODO4_CFG.STAZIONI add INVIO_RT_ISTANTANEO CHAR default 'N' not null
+alter table NODO4_CFG.STAZIONI add INVIO_RT_ISTANTANEO CHAR default 'N' not null;
```

Comment:
why this too ?

Reply:
this is a new field added in the task [PCFG-286](https://pagopa.atlassian.net/browse/PCFG-286)